src/main/groovy/io/seqera/wave/service/persistence/WaveScanRecord.groovy

```diff
@@ -38,7 +38,7 @@ import io.seqera.wave.util.StringUtils
 @EqualsAndHashCode
 @CompileStatic
 class WaveScanRecord {
-    String id
+    String scanId
```

Comment:
What's the rationale of this? I fear this will break all existing records

Reply:
now surreal does not allow to have id in record and inside the object too, so we cannot name it id
and I have tested it and everything works fine

Reply:
ok correct
the current one in surrealdb will be affected